mvlearn/embed/mvmds.py

```diff
@@ -49,6 +49,45 @@ class MVMDS(BaseEmbed):
 
     Notes
     -----
+    
+    Classical Multiview Multidimensional Scaling can be broken down into two
+    steps. The first step involves calculating the Euclidean Distance matrices, 
+    :math:`X_i`, for each of the :math:`k` views and double-centering 
+    these matrices through the following calculations:
+
+    .. math::
+        \Sigma_{i}=-\frac{1}{2}J_iX_iJ_i
+
+    .. math::
+        \text{where }J_i=I_i-{\frac {1}{n}}11_i'
+
+
+    The second step involves finding the common principal components of the
+    :math:`\Sigma` matrices. These can be thought of as multiview
+    generalizations of the principal components found in principal component
+    analysis (PCA) given several covariance matrices. The central hypothesis of
+    the common principal component model states that given k normal populations
+    (views), their :math:`p` x :math:`p` covariance matrices
+    :math:`\Sigma_{i}`, for :math:`i = 1,2,...,k` are simultaneously
+    diagonalizable as:
+
+    .. math::
+        \Sigma_{i} = QD_i^2Q^T
+
+    where :math:`Q` is the common :math:`p` x :math:`p` orthogonal matrix and
+    :math:`D_i^2` are positive :math:`p` x :math:`p` diagonal matrices. The
+    :math:`Q` matrix contains all the common principal components. The common
+    principal component, :math:`q_j`, is found by solving the minimization
+    problem:
+
+    .. math::
+        \text{Minimize} \sum_{i=1}^{k}n_ilog(q_j^TS_iq_j)
+    .. math::
+        \text{Subject to } q_j^Tq_j = 1
+
+    where :math:`n_i` represent the degrees of freedom and :math:`S_i`
+    represent sample covariance matrices.
+
```

Comment:
Is there an actual paper to add a citation for? Say https://arxiv.org/abs/1605.00286

Reply:
That's for metric MDS. This is classical mds which is effectively scaled pca

Reply:
Sorry, I meant for MVMDS in general.